src/node/command.js

```diff
@@ -137,9 +137,11 @@ function compileAll(out, sources, options) {
     traceurAPI.recursiveModuleCompileToSingleFile(out, sources, options).then(function() {
       process.exit(0);
     }).catch(function(err) {
-      var errors = err.errors || [err];
-      errors.forEach(function(err) {
-        console.error(err.stack || err);
+      if (err.name === 'MultipleErrors') {
+        err = err.errors;
+      }
+      err.forEach(function(err) {
```

Comment:
If, say a library we use, throws an error it won't be an array and the error handling will fail at `forEach`.